sw/device/lib/testing/flash_ctrl_testutils.c

```diff
@@ -53,32 +54,34 @@ bool flash_ctrl_testutils_wait_transaction_end(
         bitfield_bit32_write(error_reg, FLASH_CTRL_ERR_CODE_UPDATE_ERR_BIT,
                              codes.shadow_register_error);
   }
-  CHECK_DIF_OK(
-      dif_flash_ctrl_clear_error_codes(flash_state, output.error_code.codes));
-  return output.operation_error == 0;
+  TRY(dif_flash_ctrl_clear_error_codes(flash_state, output.error_code.codes));
+  return OK_STATUS();
```

Comment:
Now the function is returning `OK_STATUS()` but previously it was returning `output.operation_error == 0`. Check that this is correct.